src/sdk/object_sdk.js

```diff
@@ -180,8 +180,13 @@ class ObjectSDK {
         try {
             // NAMESPACE_FS HACK
             if (process.env.NAMESPACE_FS) {
+                const namespace_resource_id = _.extend({}, bucket.namespace.write_resource).resource.id;
```

Comment:
I prefer more explicit and not to make an object just for checking existence.
We can start using the new `?.` operator for these cases:
```suggestion
                const namespace_resource_id = bucket.namespace.write_resource?.resource.id;
```